src/syslog_mcp/tools/presentation/summary_formatters.py

```
@@ -0,0 +1,1109 @@
+"""
+
+from typing import Any, Dict, List, Optional
+from datetime import datetime
+
+from ...utils.logging import get_logger
+
+logger = get_logger(__name__)
+
+# Threat level threshold constants (0-100 scale)
+THREAT_CRITICAL = 80  # 80+ = Critical threat requiring immediate action
+THREAT_HIGH = 60      # 60-79 = High threat requiring prompt attention  
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)